README.md

```diff
@@ -1,8 +1,7 @@
 # FieldSerializer
 
-Welcome to your new gem! In this directory, you'll find the files you need to be able to package up your Ruby library into a gem. Put your Ruby code in the file `lib/field_serializer`. To experiment with that code, run `bin/console` for an interactive prompt.
-
-TODO: Delete this and the text above, and describe your gem
+Ruby module which allows you to specify a set of fields that you'd like
+serialized in the `#to_h` method.
```

Comment:
My main comment here is that this is only really explaining the _what_, rather than the _why_.